src/pages/patientView/PatientViewPageTabs.tsx

```diff
@@ -342,6 +359,22 @@ export function tabs(
                     pageComponent.patientViewPageStore.namespaceColumnConfig
                         .structVar
                 }
+                customDriverName={
```

Comment:
we should try to find a way to deliver these without adding properties to the table component.   are these not really part of the column configuration?